eng/PatchConfig.props

```diff
@@ -69,4 +69,9 @@ Later on, this will be checked using this condition:
     <PackagesInPatch>
     </PackagesInPatch>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.20' ">
+    <PackagesInPatch>
+      @aspnet/signalr;
```

Comment:
This part of the change looks weird because there already is a 2.1.20 block in this file. Suspect GitHub is not reporting the file conflict. Suggest rebasing on 'release/2.1' to make sure we don't hit problems when we try to get this in.

Reply:
Much better. Still weird to see more than a one line addition. Maybe GitHub is still confused. Whatever ☹️